templates/single/assignment/content.php

```diff
@@ -101,6 +101,21 @@ function tutor_assignment_convert_seconds( $seconds ) {
 				$total_mark        = tutor_utils()->get_assignment_option( get_the_ID(), 'total_mark' );
 				$pass_mark         = tutor_utils()->get_assignment_option( get_the_ID(), 'pass_mark' );
 				$file_upload_limit = tutor_utils()->get_assignment_option( get_the_ID(), 'upload_file_size_limit' );
+				$drip_type         = get_tutor_course_settings( $course_id, 'content_drip_type', 'unlock_by_date' );
+
+				if ( 'specific_days' === $drip_type ) {
+					$days         = (int) get_item_content_drip_settings( $post_id, 'after_xdays_of_enroll' );
+					$days_in_time = 60 * 60 * 24 * $days;
```

Comment:
Use constant `DAY_IN_SECONDS` instead of `60*60*24`
Change the variable `$days_in_seconds`